demonstration/harmony/docker-compose.yml

```diff
@@ -0,0 +1,310 @@
+      dockerfile: docker/web/Dockerfile_web
+      context: ../
+      args:
+        NAMESPACE: zengineering
+        TAG: latest
+    restart: always
+    # We set the pull policy to never, in order to allow usage of locally built image.
+    # e.g. if you run `make web_client_build ; make web_server_build ; make web_build`
+    # you'll have a locally built copy. If the pull_policy were set to always, docker
+    # would never use it, and always pull from remote.
+    pull_policy: never
+    ports:
+      - 5000:5000
+    environment:
+      - ZEN_ENV=${ZEN_WEB_ENV}
+      - DONT_SYNC_SOURCEMAPS=true
+      - DATABASE_URL=${POSTGRES_DB_URI}
+      - HSTS=off
+      - SSL_POLICY=Mozilla-Modern
+      - VIRTUAL_HOST=${ZEN_WEB_HOST}
+      - LETSENCRYPT_HOST=${ZEN_WEB_HOST}
+      - LETSENCRYPT_EMAIL=${ZEN_WEB_EMAIL}
+      - DRUID_HOST=${DRUID_HOST}
+    volumes:
+      - ${OUTPUT_PATH}:/data/output
+      - ${OUTPUT_PATH}/logs:/logs
+      - ${INSTANCE_CONFIG}:/zenysis/instance_config.json
+      - ${GLOBAL_CONFIG}:/zenysis/global_config.py
+      - ${UPLOADS_DIR}:/zenysis/uploads
+    command: [ "/zenysis/docker/entrypoint_web.sh" ]
+    logging: *default-logging
+    networks:
+      - harmony_network
+      - druid_network
+
+  pipeline:
+    profiles:
+      - pipeline
+    restart: "no" # Once the pipeline is done running, we exit
+    build:
+      context: ../../
+      dockerfile: demonstration/harmony/pipeline/Dockerfile
+    volumes:
+      - ./mc.config.json:/root/.mc/config.json
+      - ../..:/src/zenysis
+      # exclude these folders:
+      - /src/zenysis/venv
+      - /src/zenysis/venv_pypy3
+      - /src/zenysis/node_modules
+    networks:
+      - harmony_network
+      - druid_network
+
+  worker:
+    platform: linux/amd64
+    image: ${DOCKER_NAMESPACE}/${DOCKER_IMAGE:-harmony-web}:${DOCKER_TAG}
+    restart: always
+    pull_policy: always
+    ports:
+      - 61234:61234
+    environment:
+      - ZEN_ENV=${ZEN_WEB_ENV}
+      - ZEN_LOG_CFG=/zenysis/log/log.prod-stackdriver.cfg
+      - BROKER_URL=redis://redis:6379/0
+    volumes:
+      - ${OUTPUT_PATH}/logs:/logs
+      - ${INSTANCE_CONFIG}:/zenysis/instance_config.json
+      - ${GLOBAL_CONFIG}:/zenysis/global_config.py
+    command:
+      [
+        "celery",
+        "-A",
+        "web.background_worker.celery",
+        "worker",
+        "--beat",
+        "--loglevel=INFO"
+      ]
+    logging: *default-logging
+    networks:
+      - harmony_network
+
+  web-renderer:
+    platform: linux/amd64
+    image: zengineering/url-to-pdf-api:latest
+    restart: always
+    pull_policy: always
+    ports:
+      # minio uses 9000, so we bumped it to 9001
+      - 9001:9000
+    environment:
+      - ALLOW_HTTP=true
+      - PORT=9000
+    networks:
+      - harmony_network
+
+  redis:
+    image: redis:latest
+    restart: always
+    pull_policy: always
+    ports:
+      - 6379:6379
+    networks:
+      - harmony_network
+
+  hasura:
```

Comment:
prod hasura starts differently than local dev. is this working? 

once everything is running, you'd want to run `scripts/data_catalog/populate_query_models_from_config.py --sql_connection_string <>` against the web container. and then AQT should load.